adapter_docs/embeddings.md

```diff
@@ -0,0 +1,48 @@
+# Embeddings
+
+This latest version of the `adapter-transformers` supports dynamically adding, loading, and deleting of `Embeddings`. This section
+will give you an overview of these features. 
```

Comment:
```suggestion
Since v2.3.0, `adapter-transformers` supports dynamically adding, loading, and deleting of `Embeddings`. This section
will give you an overview of these features. 
```